ur_robot_driver/resources/ursim_driver/ursim/aux/.vol/urcap_ros_control.script

```diff
@@ -0,0 +1,180 @@
+def urcap_ros_control():
```

Comment:
Why is this moved to the robot already? This should come from our driver, right?